test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,56 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind backend service with application using bindable information specifed in CSV
+        Given Backend service "some-backend-service.v0.1.0" CSV is installed
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The application CR "csv-demo-backend" is present
```

Comment:
this is service CR not `application CR`

Reply:
I wrote it to reuse existing steps from [steps.py](https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L291)

If it's not ok then I can update the steps.py and then other tests using it.

Reply:
you can reuse the code logic, but logically the step is about adding service CR, and a reader should not be confused. You can make the step more generic as well.

Reply:
Made this step generic.